contrib/platform/src/com/sun/jna/platform/win32/COM/Wbemcli.java

```diff
@@ -137,6 +137,24 @@ public HRESULT Get(String wszName, int lFlags, VARIANT.ByReference pVal, IntByRe
             return Get(wszName == null ? null : new WString(wszName), lFlags, pVal, pType, plFlavor);
         }
 
+        public HRESULT GetMethod(String wszName, int lFlags, PointerByReference ppInSignature, PointerByReference ppOutSignature) {
+            return GetMethod(wszName == null ? null : new WString(wszName), lFlags, ppInSignature, ppOutSignature);
+        }
+
+        public HRESULT GetMethod(WString wszName, int lFlags, PointerByReference ppInSignature, PointerByReference ppOutSignature) {
+            // 20th method in IWbemClassObjectVtbl
+            return (HRESULT) _invokeNativeObject(19,
+                new Object[]{ getPointer(), wszName, lFlags, ppInSignature, ppOutSignature}, HRESULT.class);
+        }
+
+        public IWbemClassObject GetMethod(String wszName) {
+            PointerByReference ppInSignature = new PointerByReference();
+            PointerByReference ppOutSignature = new PointerByReference();
+            HRESULT res = GetMethod(wszName, 0, ppInSignature, ppOutSignature);
+            COMUtils.checkRC(res);
+            return new IWbemClassObject(ppInSignature.getValue());
```

Comment:
Would it make sense to return `IWbemClassObject[2]` and assert in the documentation, that that is an array with exactly two elements, where the first is the in and the second is the out signature?

This would also be required as the method description states (https://learn.microsoft.com/de-de/windows/win32/api/wbemcli/nf-wbemcli-iwbemclassobject-getmethod):

> Note  The caller must call IWbemClassObject::Release on the ppInSignature and ppOutSignature pointers when these objects are no longer required.

Essentially you are leaking the inSignature object.

Reply:
I assume you meant that I am leaking the outSignature object?
This overload was meant to be make the library easier to use for the common use case where usually what you would be interested in is the inSignature so I don't think we would want to return the array of IWbemClassObject.
I updated this to send NULL for the outSignature so we don't leak it.